pkg/apis/events/types.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2017 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package events
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient=true
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Event is a report of an event somewhere in the cluster. It generally denotes some state change in the system.
+type EventTwo struct {
```

Comment:
I tried patching it, but it didn't help - updated this PR with the result.

@sttts gist for you:
```
pkg/client/clientset_generated/internalclientset/typed/events/internalversion/event.go:48: events redeclared in this block
```
Once here:
```
import (
	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	types "k8s.io/apimachinery/pkg/types"
	watch "k8s.io/apimachinery/pkg/watch"
	rest "k8s.io/client-go/rest"
	events "k8s.io/kubernetes/pkg/apis/events"
	scheme "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/scheme"
)
```
and once here:
```
// events implements EventInterface
type events struct {
	client rest.Interface
	ns     string
}
```